rules/scala/versions.bzl

```diff
@@ -0,0 +1,7 @@
+# Please ensure these stay up-to-date with the versions in `MODULE.bazel` and `tests/MODULE.bazel`.
+# Unfortunately, `MODULE.bazel` files can't call `load`, so we have to copy them there.
+scala_2_12_version = "2.12.19"
```

Comment:
I don't love this and am not aware of any way around it. 

Is there any way to load variables out of the `MODULE.bazel` file?

Reply:
I don't like it either and searched around for quite a while for a solution. Here's an issue I came across:
https://github.com/bazelbuild/bazel/issues/17880

Notably:
> I don't think I'm exaggerating that much with the following claim: The availability of and reliance on `load` in `WORKSPACE` was a major contributor to its subpar performance.

There is [`include`](https://bazel.build/rules/lib/globals/module#include), which allows `MODULE.bazel` files to be segmented, but the notable limitation of `include` is that "variable bindings (such as those used for `use_extension`) are only ever visible in the file they occur in, not in any included or including files".

I also considered writing a custom extension that would create the dependency trees and `load` these versions from `scala/versions.bzl`, but extensions can't call other extensions like `@rules_jvm_external//:extensions.bzl%maven` (i.e. extension macros don't exist), so this isn't an option either.

Reply:
I came to the same conclusion after doing some Googling. Nice to have a single source of truth for rule sets while also being able to `include` for end users.

I just don't love that there isn't a way to load something from the `MODULE.bazel` file into the rest of the workspace, e.g., the scala version.

Reply:
Agreed.